infra/aws/aws-deployment-instructions.md

```
@@ -1057,21 +1154,21 @@ Four security groups. EC2 instances have **no internet-facing inbound rules** 
 | Inbound | 8088 | TCP | sg-tazama-alb | NiFi UI |
 | Inbound | 8000 | TCP | sg-tazama-alb | JupyterHub |
 | Inbound | 7619 | TCP | sg-tazama-alb | Automation Orchestrator |
 | Inbound | 8282 | TCP | sg-tazama-alb | Datalakehouse API (via ALB / tunnel) |
-| Inbound | 8282 | TCP | sg-tazama-server-b | Datalakehouse API (CMS backend direct call — bypasses ALB) |
+| Inbound | 8282 | TCP | sg-tazama-server-b | Datalakehouse API (CMS backend direct call - bypasses ALB) |
 | Inbound | 22 | TCP | sg-tazama-eice | SSH via EICE endpoint only |
 | Outbound | All | All | `0.0.0.0/0` | Image pulls, calls to Server A + B |
 
-> Solr (:8983), Ozone SCM (:9876), Ozone S3G (:9878), and Ozone Recon (:9888) are **internal-only** — no SG inbound rules; accessed exclusively via the SSH tunnel (`tunnel-server-c.ps1`).
+> Solr (:8983), Ozone SCM (:9876), Ozone S3G (:9878), and Ozone Recon (:9888) are **internal-only** - no SG inbound rules; accessed exclusively via the SSH tunnel (`tunnel-server-c.ps1`).
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)